packages/adapter-d1/README.md

```diff
@@ -1,3 +1,79 @@
 # Prisma driver adapter for Cloudflare D1
 
-**INTERNAL PACKAGE, DO NOT USE**
+Prisma driver adapter for [Cloudflare's Serverless Driver D1](https://developers.cloudflare.com/d1/). Refer to the [announcement blog post](https://prisma.io/cloudflare-d1) and our [docs](https://www.prisma.io/docs/guides/database/cloudflare-d1) for more details.
```

Comment:
"Serverless Driver" is not really the correct terminology here. We need to fix that somehow, and probably just talk about "Cloudflare D1" maybe?

Reply:
Is that specific for Cloudflare? we _are_ using that terminology in some of our other adapter readmes

Reply:
Those adapters are for "serverless drivers", which are the Npm libraries we integrate that are made specifically for serverless (And edge) environments but are called "serverless drivers" by their authors. That is not hte case for D1, that is probably just an API or similar. It's not even a package, right, it just magically exists on deployment.

Reply:
I'm happy to just refer to it as "Cloudflare's D1" or does it make more sense to call it "Cloudflare D1"?

Reply:
"Cloudflare D1", this is a product name